skimage/io/tests/test_pil.py

```diff
@@ -50,6 +50,24 @@ def test_imread_flatten():
     assert np.sctype2char(img.dtype) in np.typecodes['AllInteger']
 
 
+def test_imread_separate_channels():
+    # Test that imread returns RGBA values contiguously even when they are
+    # stored in separate planes.
+    x = np.random.rand(3, 16, 8)
+    f = NamedTemporaryFile(suffix='.tif')
+    fname = f.name
+    f.close()
+    imsave(fname, x)
+    img = imread(fname)
+    os.remove(fname)
+    assert img.shape == (16, 8, 3), img.shape
+
+
+def test_imread_multipage_rgb_tif():
+    img = imread(os.path.join(data_dir, 'multipage_rgb.tif'))
+    assert img.shape == (2, 10, 10, 3), img.shape
```

Comment:
The second argument is what gets printed if the assert fails, it is a debug statement.  I agree about `test_tifffile`.